subprojects/build-cache/src/integTest/groovy/org/gradle/caching/local/internal/DirectoryBuildCacheCleanupIntegrationTest.groovy

```diff
@@ -73,18 +73,21 @@ class DirectoryBuildCacheCleanupIntegrationTest extends AbstractIntegrationSpec
         def lastCleanupCheck = gcFile().makeOlder().lastModified()
 
         when:
-        def trashFile = cacheDir.file("0" * 32).createFile()
-        trashFile.lastModified = System.currentTimeMillis() - TimeUnit.DAYS.toMillis(MAX_CACHE_AGE) * 2
+        def newTrashFile = cacheDir.file("0" * 32).createFile()
+        def oldTrashFile = cacheDir.file("1" * 32).createFile()
+        oldTrashFile.lastModified = System.currentTimeMillis() - TimeUnit.DAYS.toMillis(MAX_CACHE_AGE) * 2
         run()
         then:
-        trashFile.assertIsFile()
+        newTrashFile.assertIsFile()
+        oldTrashFile.assertIsFile()
         assertCacheWasNotCleanedUpSince(lastCleanupCheck)
 
         when:
         lastCleanupCheck = markCacheForCleanup()
         run()
         then:
-        trashFile.assertDoesNotExist()
+        newTrashFile.assertIsFile()
```

Comment:
Why do we call those things `trashFiles`? Aren't they just build cache entries? Or we can call one `usedBuildCacheEntry` and the other one `unusedBuildCacheEntry`? WDYT?

Reply:
The name was introduced by @big-guy. I think the reason was that the files are to be deleted. Which is not completely true for `newTrashFile`, but... I don't feel very strongly about this.

Reply:
I am don't feel very strongly about this either. Your call.